src/code/sys_cfb.c

```diff
@@ -38,22 +44,43 @@ void SysCfb_Init(s32 n64dd) {
 
     // "The final address used by the system is %08x"
     PRINTF("システムが使用する最終アドレスは %08x です\n", sSysCfbEnd);
+#if ENABLE_SPLIT_FRAMEBUFFERS // split framebuffers
+    sSysCfbFbPtr[0] = 0x80700000;
+    sSysCfbFbPtr[1] = 0x80600000;
+#if ENABLE_SPLIT_ZBUFFER // and z-buffer
+    sSysCfbFbPtr[2] = 0x80500000;
+#endif
```

Comment:
This is going to break without warning if anyone tries to use it with only 4MB

Reply:
looking at this we actually kinda worked for 2 months getting 4mb mode working and im unsure if it still works